janitor/io.py

```diff
@@ -110,3 +111,66 @@ def read_commandline(cmd: str, **kwargs) -> pd.DataFrame:
     else:
         outcome = outcome.stdout
     return pd.read_csv(StringIO(outcome), **kwargs)
+
+
+def xlsx_table(
+    path: str,
+    sheetname: str,
+    table: Union[str, list, tuple] = None,
+    header: bool = True,
+) -> pd.DataFrame:
+    """
+    Returns a DataFrame of values in a table in the Excel file.
+    If the `table` argument is provided, a pandas DataFrame is returned;
+    if the `table` argument is None, or a list/tuple of names,
+    a dictionary of DataFrames is returned, where the keys of the dictionary
+    are the table names.
+
+    :param path: path to the Excel File.
+    :param sheetname: Name of the sheet
+        from which the tables are to be extracted.
+    :param table: name of a table, or list of tables in the sheet.
+    :raises ValueError: if there are not tables in the sheet.
```

Comment:
I think the `:raises ValueError:` section is conventionally placed at the end of the docstring. Could you check our convention used in the library and conform this docstring to the rest?